src/styles/components/_sandpack.scss

```diff
@@ -0,0 +1,163 @@
+// the article container
+.sandpack {
+  header {
+    color: var(--foreground-color);
+
+    h2 {
+      position: relative;
+      font-size: 1.5625rem; // 25px/16
+      font-family: Viga, sans-serif;
+      line-height: 2.125rem; // 34px/16
+      margin-inline: auto;
+
+      &::after {
+        content: "";
+        position: absolute;
+        width: 2rem;
+        height: 2.5rem;
+        background-image: url("/images/cmt_inventory_arrow.svg");
+        transform: translateX(0.5rem) rotate(-30deg);
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: cover;
+
+        @media (prefers-color-scheme: dark) {
+          filter: invert(1);
+        }
+      }
+
+      @media (min-width: $breakpoint-increase-content-font-size) {
+        font-size: map.get($font-sizes, "md");
+        line-height: 2.5rem; // 40px/16
+
+        &::after {
+          width: 2.5rem;
+          height: 3rem;
+        }
+      }
+    }
+  }
+}
+
+// because we have no control over the auto-generated sandpack class names,
+// I turned off the kebab-case linting error for certain selectors
+
+// sandpack component wrapper
+/* stylelint-disable-next-line selector-class-pattern */
+// .sp-wrapper.light.sp-c-fVPbOs-gMQIch-variant-light {
+//   height: 30rem;
+// }
+
+// component editor wrapper
+/* stylelint-disable-next-line selector-class-pattern */
+// .sp-layout.sp-c-ctabmA {
+//   height: 30rem;
+// the height of the editor can be passed into the options object on the Sandpack component!
+// }
+
+// button[title="Open in CodeSandbox"] {
+// display: none !important;
+// }
+
+// There are a TON of !important to overwrite out-of-the-box custom CSS variables
+
+// component editor
+/* stylelint-disable-next-line selector-class-pattern */
+// .sp-c-ctabmA > .sp-c-kLppIp {
+//   height: 30rem !important;
+// }
+
+.sp-code-editor {
+  transition: all 150ms ease-in-out;
+  border: 2px solid transparent;
+
+  // sandpack code editor
+  &:hover,
+  &:focus-within {
+    border: 2px solid var(--accent-color);
```

Comment:
This gets a little distracting when we set the border on `:hover`. I'd only set this for `:focus-within`.

Reply:
Should I do anything different for the hover?

Reply:
I don't think we need anything extra for the hover state. The cursor already changes to indicate text selection, which I think is enough.

Reply:
Ah yes. The cursor. Thanks!